lib/qa/authorities/linked_data/config.rb

```diff
@@ -23,27 +23,30 @@ class << self
       end
 
       attr_reader :authority_name
-      attr_reader :authority_config
 
```

Comment:
The attr_reader `authority_config` was never used.  Access was through the `auth_config` method.  Since that method was providing access to the `@authority_config` instance variable, I changed the name of the method to be `authority_config` to be more consistent and got rid of the attr_reader.